node_modules/typescript/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "typescript",
     "author": "Microsoft Corp.",
     "homepage": "https://www.typescriptlang.org/",
-    "version": "3.9.2",
+    "version": "3.6.3",
```

Comment:
This downgrades typescript as old version (v3.6.3) is listed in https://github.com/denoland/deno/blob/69552b1defcd77748de87fc4c9311b1baec1361d/tools/package.json#L10

Reply:
Hmmm... I wonder how this was generated in past commits. Obviously something is off here.

Reply:
I couldn't find documented process to update deno_third_party modules in this repo.
I manually updated typescript to v3.9.7 (copying contents from yarn install in another folder) in PR https://github.com/denoland/deno_third_party/pull/75